sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -218,6 +230,9 @@ export const DefaultCertificatePolicy: {
 export interface DeleteCertificateOperationOptions extends coreHttp.OperationOptions {
 }
 
+// @public
+export type DeleteCertificatePollerLike = PollerLike_2<PollOperationState<DeletedCertificate>, DeletedCertificate>;
```

Comment:
it feels weird that these are using the new PollerLike without adding any new state

Reply:
I'll resolve this for now.